Tools/machines/greatlakes-umich/greatlakes_v100_warpx.profile.example

```diff
@@ -0,0 +1,62 @@
+# please set your project account
+export proj=""  # change me!
+
+# remembers the location of this script
+export MY_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+if [ -z ${proj-} ]; then echo "WARNING: The 'proj' variable is not yet set in your $MY_PROFILE file! Please edit its line 2 to continue!"; return; fi
+
+# required dependencies
+module purge
+module load gcc/10.3.0
+module load cuda/12.1.1
+module load cmake/3.26.3
+module load openblas/0.3.23
+module load openmpi/4.1.6-cuda
+
+# optional: for QED support
+module load boost/1.78.0
+
+# optional: for openPMD and PSATD+RZ support
+module load phdf5
```

Comment:
@archermarx with all of the above modules loaded, what does `module av 2>&1 | grep -i phdf5` show?

Reply:
Also, with phdf5 loaded (without the cuda version of openmpi), `module av 2>&1 | grep -i phdf5 gives'

```
mpifileutils/0.10.2-arcts    phdf5/1.12.1 (L)
```

Reply:
Ah, understood. But we would like to use the cuda-aware OpenMPI. Great, Brendan has a support ticket open for that.

Reply:
Yesterday, the Great Lakes team added support for openmpi/4.1.6-cuda and phdf5/1.12.1.

```
[bstassel@gl-login3 ~]$ module purge
[bstassel@gl-login3 ~]$ module load gcc/10.3.0
[bstassel@gl-login3 ~]$ module load cuda/12.1.1
[bstassel@gl-login3 ~]$ module load cmake/3.26.3
[bstassel@gl-login3 ~]$ module load python/3.12.1
[bstassel@gl-login3 ~]$ module load openblas/0.3.23
[bstassel@gl-login3 ~]$ module load boost/1.78.0
[bstassel@gl-login3 ~]$ module load openmpi/4.1.6-cuda
[bstassel@gl-login3 ~]$ module load phdf5
[bstassel@gl-login3 ~]$
[bstassel@gl-login3 ~]$
[bstassel@gl-login3 ~]$ module li

Currently Loaded Modules:
  1) gcc/10.3.0    3) cmake/3.26.3    5) openblas/0.3.23   7) ucx/1.14.1           9) szip/2.1.1
  2) cuda/12.1.1   4) python/3.12.1   6) boost/1.78.0      8) openmpi/4.1.6-cuda  10) phdf5/1.12.1
```

Reply:
```suggestion
module load phdf5/1.12.1
```